packages/financial-templates-lib/src/price-feed/CryptoWatchPriceFeed.js

```diff
@@ -169,10 +169,14 @@ class CryptoWatchPriceFeed extends PriceFeedInterface {
 
     // 3. Check responses.
     if (!priceResponse || !priceResponse.result || !priceResponse.result.price) {
+      // TODO: Slack transport might not be able to handle long messages such as the stringified `ohlcResponse`,
```

Comment:
Good catch @nicholaspai. I've added an issue to track this: https://github.com/UMAprotocol/protocol/issues/2121

Reply:
Yeah we might want to catch these up stream in the SlackTransport for example. It could just truncate messages to the limit.

Reply:
ye I agree 100%. it should happen in the transport.